File: 09. Array/C. Modus Terbesar/c_modus_terbesar.pas
program c_modus_terbesar;
var
    arr : array[1..1000] of longint;
    N, i, num, ans : longint;
begin
    readln(N);
    for i := 1 to 1000 do arr[i] := 0;
    for i := 1 to N do begin
      read(num);
      arr[num] := arr[num]+1;
    end;
    ans := 1;
    for i := 1 to 1000 do
      if (arr[i] >= arr[ans]) then
        ans := i;
    writeln(ans);
end.

File: 09. Array/C. Modus Terbesar/c_modus_terbesar.c
#include <stdio.h>
int main(){
  int arr[1001], N, i, num, ans;
  scanf("%d", &N);
  for (i = 0; i <= 1000; i++) arr[i] = 0;
  for (i = 1; i <= N; i++){
    scanf("%d", &num);
    arr[num] += 1;
  }
  ans = 0;
  for (i = 0; i <= 1000; i++)
    if (arr[i] >= arr[ans])
      ans = i;
  printf("%d\n", ans);
  return 0;
}
